eks-anywhere-snow/Addons/Partner/Sysdig/sysdig.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: sysdig
+  namespace: sysdig
+spec:
+  chart:
+    spec:
+      chart: sysdig-deploy
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: sysdig
+        namespace: flux-system
+      version: 1.5.46
+  interval: 30s
+  url: https://charts.sysdig.com
+  values:
+    global:
+      sysdig:
+        accessKey: ${sysdigAccessKey}
```

Comment:
Did the installation work fine with this and did the installation pull `sysdig-access-key` from AWS Secrets manager store in your test?

Reply:
no, it didn't
I'm learning how to handle secrets and will update the PR

Reply:
I see that EKS-A plugins rely on external stores so I can't use default k8s secrets

Reply:
You can follow the samples like `botkube` or `kubecost` in common and partner addons folders in different installations for sample.

Reply:
Yeah @elamaran11 , thanks. That is exactly what I did and it is passing the test and the valudation